pkgs/development/compilers/dotnet/dotnet-sdk-setup-hook.sh

```diff
@@ -100,6 +101,18 @@ configureNuget() {
             -i /configuration/packageSources/__new -t attr -n value -v "$nugetSource"
             -r /configuration/packageSources/__new -v add)
 
+        if [[ -n ${keepNugetConfig-} ]] && ! xmlstarlet select -t -i "/configuration/packageSources/clear" -nl "$rootConfig"; then
+            @xmlstarlet@/bin/xmlstarlet ed --inplace \
```

Comment:
can we use `dotnet nuget add source`  for this?

Reply:
The idea was that I wanted it to be always first, but if order doesn't matter, then I guess?